installer/settings/files_main_install.nsh

```diff
@@ -89,6 +89,8 @@ IfFileExists $INSTDIR\libtiff-5.dll 0 +2
 Delete "$INSTDIR\libtiff-5.dll"
 IfFileExists $INSTDIR\libwebp-4.dll 0 +2
 Delete "$INSTDIR\libwebp-4.dll"
+IfFileExists $INSTDIR\libiconv.dll 0 +2
+Delete "$INSTDIR\libiconv.dll"
```

Comment:
This is for deleting files that are no longer used after an update. So nothing should be added to this file until we stop using the DLL.

Reply:
Thanks, didn't notice.